frontend/src/apis/rooms.api.ts

```diff
@@ -51,3 +51,10 @@ export const postParticipateIn = async (roomId: number): Promise<void> => {
     errorMessage: MESSAGES.ERROR.POST_PARTICIPATE_IN,
   });
 };
+
+export const deleteParticipateIn = async (roomId: number): Promise<void> => {
+  const res = await apiClient.delete({
+    endpoint: API_ENDPOINTS.PARTICIPATE_IN(roomId),
+    errorMessage: MESSAGES.ERROR.DELETE_PARTIPATE_IN,
+  });
+};

```

Comment:
```suggestion

export const deleteParticipateIn = async (roomId: number): Promise<void> => {
  return apiClient.delete({
    endpoint: API_ENDPOINTS.PARTICIPATE_IN(roomId),
    errorMessage: MESSAGES.ERROR.DELETE_PARTIPATE_IN,
  });
};

```

요렇게도 할 수 있을거 같은데 우선 불필요한 res 변수는 사용하지 않는게 좋을거 같아요!

Reply:
그러면 `postParticipateIn` 도 똑같이 `Promise<void>` 반환타입을 가지고 있는데,  이렇게 수정하는 게 좋을까요?